lib/ret/hub.ex

```diff
@@ -83,6 +83,8 @@ defmodule Ret.Hub do
 
     field(:allow_promotion, :boolean)
 
+    field(:member_cap, :integer)
```

Comment:
maybe `max_members`?

Reply:
also terminology is a bit rough. the list of people in the app is the 'member list', which includes lobby. maybe we should just call these "lobby members" and "room members"? (and hence, call this `max_room_members`)?

Reply:
We also have already `max_occupant_count`, which is the stats field used to track the maximum number of people who ever entered...

Reply:
ah that makes sense. room_member_cap then?

Reply:
sticking with `member_cap` since we don't prefix any other room properties with `room` except for the global `room_id`